prisma/schema.prisma

```diff
@@ -175,4 +176,20 @@ model OrderReturn {
   user            User     @relation(fields: [userId], references: [id])
   product         Product  @relation(fields: [productId], references: [id])
   order           Order    @relation(fields: [orderId], references: [id])
+}
+
+
+
+enum PaymentStatus {
+  PENDING,
+  COMPLETED,
+  FAILED,
+}
+
+enum OrderStatus {
+  PENDING,
+  PROCESSING,
+  SHIPPED,
+  DELIVERED,
+  CANCELLED,
```

Comment:
@kencas Please kindly check your enum definition here. There are matching errors on the terminal.